src/main/ml-modules/services/KoopProvider.sjs

```diff
@@ -712,14 +712,15 @@ function getObjects(req) {
   pipeline = pipeline
     .select(getSelectDef(outFields, columnDefs, returnGeometry));
 
-  const opticResult = Array.from(pipeline.result(null, bindParams))
-  const opticResultCount = opticResult.length
+  const opticResult = pipeline.result(null, bindParams)
+  const opticResultCount = Array.from(opticResult).length
 
   if(opticResultCount >= (limit+1) ){
-    return {result: opticResult.slice(0,-1), limitExceeded : true}
+    return {result:  Array.from(fn.subsequence(opticResult, 1, (opticResultCount - 1))),
```

Comment:
Since you have to use Array.from() above, I think the most efficient will be to use pop() here.

```
opticResult.pop();
return {result: opticResult, limitExceeded : true};
```